pgx-pg-sys/src/submodules/errcodes.rs

```diff
@@ -0,0 +1,1291 @@
+use std::fmt::{Display, Formatter};
+
+/// This list of SQL Error Codes is taken directly from Postgres 12's generated "utils/errcodes.h"
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub enum PgSqlErrorCode {
+    None = -1,
```

Comment:
Along those lines, ISTM like we could probably ditch this, and push users to use `Option<PgSqlErrorCode>`.

I'm also not 100% sure when one would want to use `PgSqlErrorCode::None` over `PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION` -- I guess the first is closer to `Option::None`, and the second represents error code from C.

Reply:
I'll take that `None` out now.  You're right that it SUCCESSFUL_COMPLETION is the correct choice in that case.